x-pack/plugins/security_solution/common/endpoint/generate_data.ts

```diff
@@ -466,11 +493,28 @@ export class EndpointDocGenerator {
       },
       host: this.commonInfo.host,
       process: {
+        pid: options.pid ? options.pid : this.randomN(5000),
```

Comment:
`0` is an expected value for pid. Perhaps use:
`'pid' in options ? options.pid : this.randomN(5000)`

Reply:
https://github.com/elastic/kibana/pull/69098/commits/cc5858a7d21a90e06940e530a821cd27ec080092